cfy_manager/main.py

```diff
@@ -221,6 +228,8 @@ def _only_on_brokers():
 
 
 @argh.named('add')
+@argh.arg('-c', '--config-file', action='append', default=None,
```

Comment:
feel free to `config_arg = argh.arg('-c', ...)` and then `@config_arg`, so that this doesn't have to be repeated in the 100 places

Reply:
good point, thanks